plugins/backstage-plugin-coder/docs/guides/sdk-recommendations.md

```diff
@@ -0,0 +1,557 @@
+# Using the Coder SDK
+
+This document walks you through using the Coder SDK from within Backstage.
```

Comment:
I wonder if the usage of `SDK` here might feel misleading. What do you think about rephrasing to `using the Coder API` or something similar?

Reply:
Agreed, given we don't publish a Javascript SDK this may be best. We only publish a golang SDK 👍🏼 

We should definitely change this once we publish our SDK

Reply:
I like `using the Coder API` or `making calls to the Coder API`.

Reply:
Also if we rename, the filename could be `coder-api.md` or something like that?